src/wp-includes/functions.php

```diff
@@ -8716,6 +8716,16 @@ function is_wp_version_compatible( $required ) {
 	// Strip off any -alpha, -RC, -beta, -src suffixes.
 	list( $version ) = explode( '-', $wp_version );
 
+	if ( is_string( $required ) && substr_count( $required, '.' ) > 1 && str_ends_with( $required, '.0' ) ) {
+		error_log(
+			__FUNCTION__ . '(): ' .
+			/* translators: s: version string sent to function */
+			sprintf( __( '`%s` Not a valid WordPress version string.' ), $required )
```

Comment:
It would be helpful to present both what was passed in and what the result was. What do you think of this instead?

"Invalid version 6.0.0 passed to `is_wp_version_compatible`, 6.0 is assumed."